config.json

```diff
@@ -256,6 +256,22 @@
         "math"
       ]
     },
+    {
+      "slug": "kindergarten-garden",
+      "uuid": "0b92ffee-c092-4ab1-ad78-76c8cf80e1b5",
+      "core": false,
+      "unlocked_by": "matrix",
```

Comment:
I don't believe the matrix exercise is part of the Kotlin track yet - can you specify a valid exercise please?